File: Chapter3.py
import cv2
import numpy as np

#Resizing and Cropping

img = cv2.imread("lambo.png")
print(img.shape)
imgResize = cv2.resize(img,(300,300))
imgCrop = img[0:462,200:300]
cv2.imshow("Original Image", img)
cv2.imshow("Resize Image", imgResize)
cv2.imshow("Cropped Image", imgCrop)
cv2.waitKey(0)

File: Chapter3.cpp
#include <opencv2/imgcodecs.hpp>
#include <opencv2/highgui.hpp>
#include <opencv2/imgproc.hpp>
#include <iostream>

using namespace cv;
using namespace std;
//
//int main() {
//	string path = "Resources/test.png";
//	Mat img = imread(path);
//	Mat imgCrop, imgResize;
//
//	resize(img, imgResize, Size(640 , 480));
//	imshow("Image Resize", imgResize);
//	imshow("Original", img);
//	waitKey(0);
//
//	return 0;
//}



int main() {
	string path = "Resources/test.png";
	Mat img = imread(path);
	Mat imgCrop;

	Rect roi(200, 40, 300, 350);
	imgCrop = img(roi);
	imshow("Original", img);
	imshow("Image Crop", imgCrop);
	waitKey(0);

	return 0;
}